.eslintignore

```diff
@@ -49,5 +49,6 @@ yarn-error.log*
 ## Shared between linters
 #########################
 
-.git/
-.yarn/
+/.git/
+/.yarn/
+/public/iframes/
```

Comment:
Prevents linting of `public/iframes/iframe-resizer.content-window.min.js` and `public/iframes/iframe-resizer.min.js`. Leading slash indicates that all paths descend from repo root (we don’t expect `/foo/bar/.yarn/` etc.)